### PR TITLE
Bring back the Hall of Fame

### PR DESCRIPTION
**Pull request by gzm0**

### CODE REVIEW

**Comment by gzm0** [comment]
Review by @sjrd

File: community/halloffame.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+Prior to Node.js 0.10.29, comparing `Int.MaxValue` with `Int.MinValue` could yield a wrong result.
+Notably:
+
+{% highlight javascript %}
+$ node
+> 2147483647 > -2147483648
+false
+{% endhighlight %}
+
+This [Node.js bug][1] was discovered by a [test][2] in Scala's partest suite that we run for Scala.js.
```
Comment (sjrd) [inline community/halloffame.md]: We usually use the `[Node.js bug](url)` notation, so that we can also read the .md reasonably well.

Comment (gzm0) [inline community/halloffame.md]: Hahaha, ok. I used this notation so you can read the .md reasonably well :P I'll change it.

File: community/halloffame.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+class Foo {}
+Foo.x = 5;
+class Bar extends Foo{}
+Bar.x // -> 5
+{% endhighlight %}
+
+While this behavior is specified in ECMAScript 6, there is no reliable way to reproduce it in pure ECMAScript 5.
+This would have essentially meant, that React would not work on some fully ECMAScript 5 compliant platforms, a notable example being Internet Explorer 10 and earlier.
+
+After the discussion on the [Scala.js bug][5], the React team changed the design and is using a [non-static method now][6].
```
Comment (sjrd) [inline community/halloffame.md]: now using

File: community/halloffame.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+
+{% highlight javascript %}
+class Foo {}
+Foo.x = 5;
+class Bar extends Foo{}
+Bar.x // -> 5
+{% endhighlight %}
+
+While this behavior is specified in ECMAScript 6, there is no reliable way to reproduce it in pure ECMAScript 5.
+This would have essentially meant, that React would not work on some fully ECMAScript 5 compliant platforms, a notable example being Internet Explorer 10 and earlier.
```
Comment (sjrd) [inline community/halloffame.md]: no `,` between "meant" and "that"?

**Comment by sjrd** [comment]
That's all.

**Comment by gzm0** [comment]
Updated.

**Comment by sjrd** [comment]
LGTM

**Pull request merged**